doc/py_tutorials/py_feature2d/py_matcher/py_matcher.markdown

```diff
@@ -53,8 +53,8 @@ import numpy as np
 import cv2 as cv
 import matplotlib.pyplot as plt
 
-img1 = cv.imread('box.png',0)          # queryImage
-img2 = cv.imread('box_in_scene.png',0) # trainImage
+img1 = cv.imread('box.png',cv.IMREAD_UNCHANGED)          # queryImage
+img2 = cv.imread('box_in_scene.png',cv.IMREAD_UNCHANGED) # trainImage
```

Comment:
0 is `IMREAD_GRAYSCALE`

Reply:
Thank you for your review. I fixed the mode of imread.